modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java

```diff
@@ -35,13 +38,14 @@
     private static final long serialVersionUID = 0L;
 
     /** Default console URI. */
-    private static final String DFLT_CONSOLE_URI = "http://localhost:3000";
+    private static final String DFLT_CONSOLE_URIS = F.isEmpty(IgniteProperties.get("ignite.management.console.uris"))
```

Comment:
Don't we have `ifPresent()`/`orElse()` methods?

Reply:
No